### PR TITLE
Dark mode fixes

### PR DESCRIPTION
**Pull request by teolemon**
image

### CODE REVIEW

**Comment by VaiTon** [comment]
I would like to use attributes instead of colors. Any reason on using them?

**Comment by teolemon** [comment]
no reason, I just didn't know what I was doing :-)

**Comment by teolemon** [comment]
Also, the cards are still very ugly. I just changed the color to highlight where the logic was.

**Comment by teolemon** [comment]
@VaiTon how should I go about changing to Attributes ?

File: app/src/main/res/drawable-night/ic_filter_list_black_24dp.xml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    android:width="24dp"
+    android:height="24dp"
+    android:viewportHeight="24.0"
+    android:viewportWidth="24.0">
+    <path
+        android:fillColor="#FFFFFF"
```
Comment (teolemon) [inline app/src/main/res/drawable-night/ic_filter_list_black_24dp.xml:7]:
```suggestion
        android:fillColor="@color/white"
```

File: app/src/main/res/layout-night/activity_splash.xml [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+    xmlns:tools="http://schemas.android.com/tools">
+
+    <data>
+
+    </data>
+
+    <androidx.constraintlayout.widget.ConstraintLayout
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:background="#000000">
```
Comment (teolemon) [inline app/src/main/res/layout-night/activity_splash.xml:29]:
```suggestion
        android:background="@color/md_black_1000">
```

File: app/src/main/res/drawable-night/textview_full.xml [hunk 3]
```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<shape xmlns:android="http://schemas.android.com/apk/res/android"
+    android:shape="rectangle" android:padding="10dp" >
+    <solid android:color="@color/md_black_1000"/>
+    <stroke
+        android:width="0.5dp"
+        android:color="@color/grey_50" />
```
Comment (teolemon) [inline app/src/main/res/drawable-night/textview_full.xml:7]:
```suggestion
        android:color="@color/md_black_1000" />
```

File: app/src/main/res/layout/activity_splash.xml [hunk 4]
```diff
@@ -25,7 +25,8 @@
 
     <androidx.constraintlayout.widget.ConstraintLayout
         android:layout_width="match_parent"
-        android:layout_height="match_parent">
+        android:layout_height="match_parent"
+        android:background="#000000">
```
Comment (teolemon) [inline app/src/main/res/layout/activity_splash.xml:29]:
```suggestion
        android:background="@color/white">
```

Comment (teolemon) [inline app/src/main/res/layout/activity_splash.xml:29]:
```suggestion
        android:background="@color/white">
```

**Comment by teolemon** [comment]
thank you 👍

**Pull request merged**